tests/experimental/metric_repository/test_metric_list_metric_retriever.py

```diff
@@ -257,24 +270,10 @@ def test_column_metrics_not_returned_if_column_types_missing(mocker: MockerFixtu
         computed_metrics,
         aborted_metrics,
     )
-    mock_batch = mocker.Mock(spec=Batch)
-    mock_batch.id = "batch_id"
-    mock_validator.active_batch = mock_batch
-
-    metric_retriever = MetricListMetricRetriever(context=mock_context)
 
-    mock_batch_request = mocker.Mock(spec=BatchRequest)
-
-    mocker.patch(
-        f"{MetricListMetricRetriever.__module__}.{MetricListMetricRetriever.__name__}._get_numeric_column_names",
-        return_value=[],
-    )
-    mocker.patch(
-        f"{MetricListMetricRetriever.__module__}.{MetricListMetricRetriever.__name__}._get_timestamp_column_names",
-        return_value=["timestamp_col"],
-    )
+    patch_get_numeric_column_names_with(mocker, ["timestamp_col"])
```

Comment:
Should this be patch _get_timestamp_column_names instead of numeric column names?

Reply:
And also add a patch_get_numeric_column_names_with as well?

Reply:
Like this:
```suggestion
    patch_get_numeric_column_names_with(mocker, [])
    patch_get_timestamp_column_names_with(mocker, ["timestamp_col"])
```

Reply:
yes good catch, updated in 29e8fb319